scene/2d/camera_2d.h

```diff
@@ -86,6 +86,7 @@ class Camera2D : public Node2D {
 
 	Point2 camera_screen_center;
 	bool _is_editing_in_editor() const;
+	void _project_settings_changed();
```

Comment:
Should this be wrapped in `TOOLS_ENABLED`? I wasn't sure.

Reply:
I think you can `#ifdef` the entire method and its implementation as done in https://github.com/godotengine/godot/pull/90049#issuecomment-2115979739 for node configuration warnings. This should reduce binary size ever so slightly.